raiden_libs/network/matrix/client.py

```diff
@@ -128,6 +128,7 @@ def __init__(
         self.sync_thread = None
         self._handle_thread = None
         self._post_hook_func: Optional[Callable[[str], None]] = None
+        self._first_sync = True
```

Comment:
May I ask to, instead of an explicitly set boolean, use the truthness of `sync_token` as indicative of the first sync?

Reply:
Doesn’t work unfortunately since I need this at the end of handle_response and by that time the sync token has already been set on the instance.